handler.go

```diff
@@ -153,6 +169,57 @@ func (h HandlerOpts) GetSigningKeyFallback() string {
 	return *h.SigningKeyFallback
 }
 
+// GetAPIOrigin returns the host to use for sending API requests
+func (h HandlerOpts) GetAPIOrigin() string {
+	if h.isDev() {
+		return devServerOrigin
```

Comment:
Users should still be able to override the Dev Server origin with the option or env var